Used https://gist.github.com/shivanshu1333/6b8000749d9af8a17949ce20f5ed2598 with https://github.com/kubernetes/klog/tree/main/hack/tools/logcheck on Kubernetes repo

output:
```
Running structured logging static check on migrated packages
/Users/shivashr/opensource/prow/go/src/k8s.io/kubernetes/pkg/proxy/healthcheck/service_health.go:59:3: Additional arguments to ErrorS should always be Key Value pairs. Please check if there is any key or value missing.
/Users/shivashr/opensource/prow/go/src/k8s.io/kubernetes/pkg/controller/volume/pvcprotection/pvc_protection_controller.go:353:2: Additional arguments to InfoS should always be Key Value pairs. Please check if there is any key or value missing.
Please fix above failures. You can locally test via:
hack/verify-structured-logging.sh
```
Fixed structured logging migration in mentioned files

--- a/pkg/controller/volume/pvcprotection/pvc_protection_controller.go
+++ b/pkg/controller/volume/pvcprotection/pvc_protection_controller.go
@@ -350,7 +350,7 @@ func (c *Controller) pvcAddedUpdated(obj interface{}) {
 		utilruntime.HandleError(fmt.Errorf("couldn't get key for Persistent Volume Claim %#v: %v", pvc, err))
 		return
 	}
-	klog.V(4).InfoS("Got event on PVC", "key", key)
+	klog.V(4).InfoS("Got event on PVC", "pvc", klog.KObj(pvc))
 
 	if protectionutil.NeedToAddFinalizer(pvc, volumeutil.PVCProtectionFinalizer) || protectionutil.IsDeletionCandidate(pvc, volumeutil.PVCProtectionFinalizer) {
 		c.queue.Add(key)

